mods/d2k/weapons/other.yaml

```diff
@@ -60,11 +60,10 @@ WormJaw:
 	ReloadDelay: 10
 	InvalidTargets: Structure, Infantry
 	Range: 1c512
-	Warhead@1Dam: SpreadDamage
+	Warhead@1Dam: SpreadDamage # HACK: The warhead is needed for targeting
```

Comment:
It would be slightly less hacky if this used `TargetDamageWarhead`, but that can be done in #12605.